config/initializers/sidekiq.rb

```diff
@@ -1,4 +1,4 @@
-redis_jobs_url = ENV.fetch("OFN_REDIS_JOBS_URL", "redis://localhost:6381/0")
+redis_jobs_url = ENV.fetch("OFN_REDIS_JOBS_URL", "redis://localhost:6379/0")
```

Comment:
I know this is currently a bit annoying in dev, but we can't merge this change at the moment because it would break things in production and staging.

This is a good example of why we'll be adding Dotenv soon (which will help with these issues): https://github.com/openfoodfoundation/openfoodnetwork/issues/7745

Reply:
@Matt-Yorkley I didnt actually mean to push it. I think it got in there when I merged remote with local. :sweat_smile:

Reply:
I'll just change this, and do a squish. :ok_hand: